extensions/zerion/src/shared/utils.ts

```diff
@@ -0,0 +1,161 @@
+import { groupBy } from "lodash";
```

Comment:
```suggestion
import groupBy from "lodash/groupBy.js";
```

You can import the `groupBy` in this way to get a smaller distribution file size.